app/mailers/activity_mailer.rb

```diff
@@ -22,8 +23,7 @@ def email(recipient, object, view_path)
   def mail(options)
     template = options.delete(:template)
     if template
-      prepend_view_path(File.dirname(template))
```

Comment:
I've removed prepend_view_path and now this mailer only supports relative path ( it only supports absolute path before )